### PR TITLE
ServerListPingEvent online player setter

### PR DESCRIPTION
**Pull request by Verficon2**
This setter allows people to spoof the amount of people that are online.
Implementation is done here: https://github.com/Bukkit/CraftBukkit/pull/902

### CODE REVIEW

File: src/main/java/org/bukkit/event/server/ServerListPingEvent.java [hunk 1]
```diff
@@ -60,13 +60,22 @@ public int getNumPlayers() {
...
+     * @return the maximum number of player
      */
     public int getMaxPlayers() {
         return maxPlayers;
     }
 
     /**
+     * Set the number of players sent.
+     *
+     * @param numPlayers the maximum number of player
```
Comment (ScoreUnder) [inline src/main/java/org/bukkit/event/server/ServerListPingEvent.java:72]: "the maximum number of player" - this is not the maximum number, this is the number of connected players

File: src/main/java/org/bukkit/event/server/ServerListPingEvent.java [hunk 2]
```diff
@@ -60,13 +60,22 @@ public int getNumPlayers() {
     /**
      * Get the maximum number of players sent.
      *
-     * @return the the maximum number of player
+     * @return the maximum number of player
```
Comment (ScoreUnder) [inline src/main/java/org/bukkit/event/server/ServerListPingEvent.java:63]: "player" should be plural if you're going to correct the typo

**Comment by md-5** [comment]
Oh yay another one of these. Its been said a few times that its not happening iirc.

**Comment by Wolvereness** [comment]
https://github.com/Bukkit/CraftBukkit/pull/778

**Pull request closed without merging**